number-store-list-table.php

```diff
@@ -150,6 +160,12 @@ public function get_columns() {
 		if ( ! isset( WPO_WCPDF()->settings->debug_settings['calculate_document_numbers'] ) ) {
 			unset( $columns['calculated_number'] );
 		}
+
+		if( ! isset( $_GET['table_name'] ) ||
+			( isset( $_GET['table_name'] ) && 'wp_wcpdf_invoice_number' === $_GET['table_name'] )
```

Comment:
I think for readable purposes, this line should be moved to `:164`, that is, the whole condition in a single line. Also, remember to add a space after the `if` statement.